tests/conftest.py

```diff
@@ -452,3 +453,31 @@ class AsyncMock(Mock):
 
     async def __call__(self, *args: Any, **kwargs: Any) -> Any:
         return super().__call__(*args, **kwargs)
+
+
+def raise_on_unexpected_train(f) -> Callable:
+    @functools.wraps(f)
+    def decorated(*args, **kwargs):
+        if os.environ.get("RAISE_ON_TRAIN") == "True":
+            raise ValueError("Training called and RAISE_ON_TRAIN is set.")
```

Comment:
How about adding a link to the readme section and / or telling developers how to fix this?